source/_integrations/withings.markdown

```diff
@@ -40,9 +40,11 @@ Once saved, the "Client Id" and "Consumer Secret" fields will be populated. You
     withings:
       client_id: CLIENT_ID
       client_secret: CONSUMER_SECRET
+      use_webhook: true
     ```
 - Confirm your YAML configuration is valid by using the `Check Config` tool (see note).
     - Note: In order for "Check Configuration" to be visible, you must enable "Advanced Mode" on your user profile. The "Check Configuration" tool can be found by clicking "Configuration" from the sidebar (cog icon) and then clicking "Server Control".
+- make sure your External URL (under Configuration > General) is set to your Nabu Casa Cloud instance URL
```

Comment:
This is incorrect. You'd never set the Nabu Casa Cloud URL in the external URL.

Reply:
that was the only thing that worked for me, otherwise I got errors setting up webhooks in the log and my entities didn't update. it's possible there's a bug in the component that causes this. the soloution i found for this was to set the external URL based on this thread: https://github.com/home-assistant/core/issues/37290

Reply:
The integration does not implement cloud support, which is part of the issue.

The external URL should NOT contain the Nabu Casa URL. The Nabu Casa URL is used automatically when the external URL isn't set, the cloud is activated and a external URL was requested.

A better solution would be for Withings to actually implement support for Cloud hooks, as most other integrations do.

Reply:
ok, I'll cancel this then.